SampleApps/CarthageTest/Cartfile

```diff
@@ -1 +1 @@
-github "braintree/braintree-ios-drop-in"
\ No newline at end of file
+git "file:///Users/scannillo/bt/braintree-ios-drop-in" "spm-support"
```

Comment:
Will this be reverted to use the github repo in a later PR?

Reply:
Good catch - this looks like a remnant of our `rake` task when we were doing some local testing.